dogspace/src/spot/scripts/camera_sub.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import roslib
+#roslib.load_manifest('my_package')
+import sys
+import rospy
+import cv2
+from std_msgs.msg import String
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge, CvBridgeError
+
+class image_converter:
+
+  def __init__(self):
+    #self.image_pub = rospy.Publisher("image_topic_2",Image)
+    self.counter = 0
+    self.bridge = CvBridge()
+    self.image_sub = rospy.Subscriber("image_raw",Image,self.callback)
+
+  def callback(self,data):
+    try:
+      cv_image = self.bridge.imgmsg_to_cv2(data, "bgr8")
+      cv2.imwrite('/media/mohamed/C03CCDB43CCDA62E1/tutorials/Innopolis/1st/FRP/HA/Robot_Dog/images/image'+str(self.counter)+'.png', cv_image)
```

Comment:
hardcode that will not work on my laptop

Reply:
so, one of the best practices is not to use a specific directory for me, just a general one ?

Reply:
use relative paths, not absolute

Reply:
edited

Reply:
at least like this, yes. The best way like: getCurrentPath() inside the script and add /images.